README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
+    - Manages wizard related commands for apps, locations, cloud providers, server plans
+
 USAGE:
-   concerto wizard command [command options] [arguments...]
+    command [command options] [arguments...]
+
 COMMANDS:
-   apps         Provides information about apps
-   cloud_providers  Provides information about cloud providers
-   locations        Provides information about locations
-   server_plans     Provides information about server plans
-   help, h      Shows a list of commands or help for one command
-OPTIONS:
-   --help, -h   show help
+     apps             Provides information about apps
+     cloud_providers  Provides information about cloud providers
+     locations        Provides information about locations
+     server_plans     Provides information about server plans
+...
 ```
 
-Concerto Wizard lets you select the application layer, the location, the cloud provider for that location, and finally the hostname. Concerto Wizard takes care of the details.
-
+IMCO CLI Wizard lets you select the application layer, the location, the cloud provider account for that location, and finally the hostname. IMCO CLI Wizard takes care of the details.
 
-If you haven't configured you cloud provider account yet, you can do it from the Concerto Web UI, or using `concerto settings cloud_accounts` commands
+If you haven't configured your cloud provider accounts yet, you can do it from the IMCO's Web UI, or using `concerto settings cloud_accounts` commands
 
 ### Wizard Use Case
-Let's type concerto wizard apps list to check what servers can I instantiate using Concerto wizard
-```
-concerto wizard apps list
-ID                         NAME              FLAVOUR REQUIREMENTS       GENERIC IMAGE ID
-53f0f126d8a5975a1c000425   Ubuntu 12.04      {}                         53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c000426   Windows 2008 R2   {"memory":2048}            53f0f0e3d8a5975a1c000071
-53f0f126d8a5975a1c000427   BigBlueButton     {"memory":2048,"cpus":2}   5409b147d8a597d977000001
-53f0f126d8a5975a1c000428   Joomla            {"memory":1024}            53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c000429   Magento           {"memory":1024}            53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c00042a   MongoDB           {}                         53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c00042b   Wordpress         {"memory":2048}            53f0f0e3d8a5975a1c000070
-5409b197d8a597d977000010   Docker            {}                         5409b147d8a597d977000001
-```
-You can choose whatever application/stack is fine for your purpose, we choose Wordpress. Take note of the application identifier, `53f0f126d8a5975a1c00042b` for Wordpress.
-We will also need the location where we want our server to be instantiated. Execute concerto wizard locations list to get the possible locations and its identifier.
+
+Let's type `concerto wizard apps list` to check what servers can I instantiate using IMCO CLI wizard.
```

Comment:
I'd type ...`to check what applications we can instantiate as cloud servers using`...